upstream-community-operators/synopsys/blackducks.crd.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  creationTimestamp: 2019-03-04T15:38:44Z
```

Comment:
The CRDs shouldn't have the deployment info in them, so things like the creationTimestamp, link, version, etc. should all be removed from the actual definition and left for OpenShift to fill out when it is created.